.github/workflows/ci.yaml

```diff
@@ -12,18 +12,20 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.operating-system }}
 
     strategy:
       matrix:
+        operating-system: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
not sure we need macos, but the only argument I have is the increased time of the build. Is there anything we get from running the build on macos?

up to you, minor for me

Reply:
IDK, let's remove it.